planning/mission_planner/README.md

```diff
@@ -149,6 +153,46 @@ To calculate `route_lanelets`,
 
 `create route sections` extracts `primitives` from `route_lanelets` for each route section with the route handler, and creates route sections.
 
+### Rerouting
+
+Reroute here means changing the route while driving. Unlike route setting, it is required to keep a certain distance from vehicle to the point where the route is changed.
+
+![rerouting_safety](./media/rerouting_safety.svg)
+
+And there are three use cases that require reroute.
+
+- Route change API
+- MRM API
+- Modify Goal
+
```

Comment:
Could you please avoid using the word `MRM API` to avoid confusion? Route change API is right, but MRM has use cases other than API. For example:

- Route change API 
- Emergency route
- Modify goal / Goal modification (noun for consistency)

Reply:
thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/3356/commits/bbb4c917a62bece4a37478b4758e5360fafc4ab8

Reply:
thanks fixed in https://github.com/autowarefoundation/autoware.universe/pull/3356/commits/d790b58ac371ff7614162ff332d0f4c33aff8dd7